src/Layout/index.js

```diff
@@ -23,6 +23,25 @@ if (TEST_ADA) {
   axe(React, ReactDOM, 1000);
 }
 
+const scrollToTop = () =>
+  document.querySelector(`.${OVERFLOW_CONTAINER_CLASS}`).scrollTo(0, 0);
+
+const onLoad = () => {
+  const hash = window.location.hash;
+
+  if (!hash) {
+    scrollToTop();
+  } else {
+    setTimeout(() => {
```

Comment:
This looks like a hack. Why are we needing this?

Reply:
Yeah, it is. I looked into it for about 45 minutes to try and find a clean solution, and I'm sure one exists. Basically, by the time useEffect runs on Layout (I also tried on DocItem) the content hasn't fully loaded yet. Which means that if we scroll the y coordinate that we're really supposed to be at can be off, sometimes by as much as 1000 pixels at the bottom of a long page.

I would love to look into it further and find a clean solution, but I just didn't want to eat up a lot of time finding one.

Reply:
What's routing look like? They using react router? If so let's use a lib like https://github.com/rafgraph/react-router-hash-link/blob/master/src/index.js

Reply:
After further discussion we have decided to open an issue with docusaurus https://github.com/facebook/docusaurus/issues/3109